container/stream/streams.go

```diff
@@ -119,30 +119,35 @@ func (c *Config) CopyToPipe(iop *cio.DirectIO) {
 	ctx := context.TODO()
 
 	c.dio = iop
-	copyFunc := func(w io.Writer, r io.ReadCloser) {
+	copyFunc := func(name string, w io.Writer, r io.ReadCloser) {
 		c.wg.Add(1)
 		go func() {
 			if _, err := pools.Copy(w, r); err != nil {
-				log.G(ctx).Errorf("stream copy error: %v", err)
+				log.G(ctx).WithFields(log.Fields{"stream": name, "error": err}).Error(err)
```

Comment:
Looks like this now logs the error twice (once as log message, and once as `error` attribute); we can probably just keep the original log message;

```go
log.G(ctx).WithFields(log.Fields{"stream": name, "error": err}).Error("stream copy error")
```


That would show as something like;

```console
ERROR[2024-08-15T09:45:03.587172878Z] stream copy error                 error="something bad happened" stream="stdout"
```